pubsub/drivertest/drivertest.go

```diff
@@ -291,6 +295,9 @@ func testSendReceiveTwo(t *testing.T, newHarness HarnessMaker) {
 		t.Fatal(err)
 	}
 	defer h.Close()
+	if !h.SupportsMultipleSubscriptions() {
+		t.Skip("multiple subscriptions to a topic not supported")
```

Comment:
What will happen if someone actually tries to create a second subscription to an SQS queue? Do we return an informative error?

Reply:
No; I don't think we can tell. The behavior would be the same as if they created a subscription in one server and another subscription in another server -- each subscription would get ~half of the messages.